pf9-addons/pf9-addon-operator/tooling/manifests/pf9-addon-operator/pf9-addon-operator-deployment.yaml

```diff
@@ -30,9 +30,6 @@ spec:
           limits:
```

Comment:
why not delete the file if addon-operator is not getting deployed anyway?

Reply:
Didn't want to lose the addon-operator YAML changes, incase we decide to go to route of having nodelet create the Addon objects. 

I can git rm if you want

Reply:
Yes, lets remove it. This YAML is going to be present in pf9-kube repo in case needed.